src/app/admin/admin.component.html

```diff
@@ -138,12 +138,12 @@
         <button
           *ngIf="!submittingGraylistUpdate"
           (click)="updateProfileModerationLevel('graylist')"
-          class="btn btn-secondary fs-15px ml-5px"
+          class="btn btn-dark fs-15px ml-5px"
```

Comment:
don't love that this is the same color as the blacklist button -- is there any way we can differentiate these buttons?

Reply:
Yes, what color? 👍

Reply:
ideally a gray-like color!